### PR TITLE
added default 0 to balance value accessing

### LINKED ISSUE #465: use default 0 for undefined balance
https://sentry.io/brave-software/ledger-production/issues/615457752/?query=is%3Aunresolved

### PR DESCRIPTION
**Pull request by maikelmclauflin**
fixes #465

### CODE REVIEW

File: bat-utils/lib/runtime-wallet.js [hunk 1]
```diff
@@ -346,8 +351,18 @@ Wallet.providers.uphold = {
     }
 
     const altScale = this.currency.alt2scale(info.altcurrency)
-    const balanceProbi = new BigNumber(cardInfo.balance || 0).times(altScale)
-    const spendableProbi = new BigNumber(cardInfo.available || 0).times(altScale)
+    const { error } = Joi.validate(cardInfo, cardInfoSchema)
+    if (error) {
+      this.runtime.captureException(error, {
```
Comment (evq) [inline bat-utils/lib/runtime-wallet.js]: would it be more useful to check this inside bat-utils where we have more context?

Comment (maikelmclauflin) [reply bat-utils/lib/runtime-wallet.js]: Sorry, i don't understand what more context we would have. I'm happy to, I just don't understand the reason.

Comment (evq) [reply bat-utils/lib/runtime-wallet.js]: we have the full response from uphold

Comment (maikelmclauflin) [reply bat-utils/lib/runtime-wallet.js]: yeah, we have the response in this function. so that means we have more context, no?

Comment (evq) [reply bat-utils/lib/runtime-wallet.js]: i'm blind :)

**Pull request merged**